test/rosutil/symdebug.rkt

```diff
@@ -0,0 +1,28 @@
+#lang rosette/safe
+
+(require rosutil
+         rackunit)
+
+(test-case "term-length basic input types"
+  (check-equal? (term-length 5) 0)
+  (define-symbolic* x integer?)
+  (check-equal? (term-length x) 1)
+  (define e (+ x 5))
+  (check-equal? (term-length e) 3)
```

Comment:
I don't see why `(term-length 5)` should be 0, while `(term-length (+ x 5))` should be 3 (like if constants are not counted when they are bare constants, why are they counted when part of an expression?)